docs/pages/management/guides/elastic-stack.mdx

```diff
@@ -0,0 +1,729 @@
+    password => "ELASTICSEARCH_PASSPHRASE"
+    template_name => "audit-events"
+    template => "/home/audit-events.json"
+    index => "audit-events-%{+yyyy.MM.dd}"
+    template_overwrite => true
+  }
+}
+```
+
+In the `input.http` section, update `ssl_certificate` and
+`ssl_certificate_authorities` to include the locations of the server certificate
+and certificate authority files that the `teleport-event-handler configure`
+command generated earlier. 
+
+Logstash will authenticate client certificates against the CA file and present a
+signed certificate to the Teleport Event Handler plugin.
+
+Edit the `ssl_key` field to include the path to the `pkcs8.key` file we
+generated earlier.
+
+In the `output.elasticsearch` section, edit the following fields depending on
+whether you are using Elastic Cloud or your own Elastic Stack deployment:
+
+<Tabs>
+<TabItem label="Elastic Cloud">
+
+Assign `cloud_auth` to a string with the content `teleport:PASSWORD`, replacing
+`PASSWORD` with the password you assigned to your `teleport` user earlier.	
+
+Visit `https://cloud.elastic.co/deployments`, find the "Cloud ID" field, copy
+the content, and add it as the value of `cloud_id` in your Logstash pipeline
+configuration. The `elasticsearch` section should resemble the following:
+
+```ruby
+  elasticsearch {
+    cloud_id => "CLOUD_ID"
+    cloud_auth => "teleport:PASSWORD" 
+    template_name => "audit-events"
+    template => "/home/audit-events.json"
+    index => "audit-events-%{+yyyy.MM.dd}"
+    template_overwrite => true
+  }
+```
+
+</TabItem>
+<TabItem label="Self-Hosted">
+
+Assign `hosts` to a string indicating the hostname of your Elasticsearch host. 
+
+Assign `user` to `teleport` and `password` to the passphrase you created for
+your `teleport` user earlier.
+
+The `elasticsearch` section should resemble the following:
+
+```ruby
+  elasticsearch {
+    hosts => "elasticsearch.example.com"
+    user => "teleport" 
+    password => "PASSWORD" 
+    template_name => "audit-events"
+    template => "/home/audit-events.json"
+    index => "audit-events-%{+yyyy.MM.dd}"
+    template_overwrite => true
+  }
+```
+
+</TabItem>
+</Tabs>
+
+Finally, modify `template` to point to the path to the `audit-events.json` file
+you created earlier. 
+
+Because the index template we will create with this file applies to indices
+with the prefix `audit-events-*`, and we have configured our Logstash pipeline
+to create an index with the title `"audit-events-%{+yyyy.MM.dd}`, Elasticsearch
+will automatically index fields from Teleport audit events.
+
+### Disable the Elastic Common Schema for your pipeline
+
+The Elastic Common Schema (ECS) is a standard set of fields that Elastic Stack
+uses to parse and visualize data. Since we are configuring Elasticsearch to
+index all fields from your Teleport audit logs dynamically, we will disable the
+ECS for your Logstash pipeline.
+
+On the host where you are running Logstash, edit `/etc/logstash/pipelines.yml`
```

Comment:
When testing with the logstash provided by my apt repositories (Pop_OS, so likely Ubuntu's package), there existed a `main` pipeline by default:

```
- pipeline.id: main
  path.config: "/etc/logstash/conf.d/*.conf"
```

Even though the only file in `/etc/logstash/conf.d/` was `teleport-audit.conf`, It would create the main pipeline and bind to the port `9601`, causing a conflict.

Not sure what the best resolution is. I assume (going on nothing) the best options are to either disable the main pipeline (as I did), or change the port our pipeline runs on.